include/exadg/fluid_structure_interaction_precice/driver_fluid.h

```diff
@@ -257,7 +257,10 @@ class DriverFluid : public Driver<dim, Number>
 
     // structure to ALE
     {
+      // Declare some data structures
       std::vector<Point<dim>> quadrature_point_locations;
+      auto exadg_terminal = std::make_shared<InterfaceCoupling<dim, dim, Number>>();
```

Comment:
I see, this works since we have scopes for ALE/fluid/structure interface coupling. We could still think about making this more clear by adding `ale` etc. to the variable names, but this is a matter of taste and not so important right now.

Reply:
Adjusted it. I thought the code might be more readable if we have more uniform names.